Add notice about dark and light theme to PR's template (#3656)
rainbow-me/rainbow: .github/pull_request_template.md

Fixes RNBW-####
Figma link (if any):

## What changed (plus any additional context for devs)

## Screen recordings / screenshots

> Screen recordings can also be helpful for showing reviewers what to test for.

Reviewer: ```suggestion
/* Screen recordings can also be helpful for showing reviewers what to test for. */
```

## What to test

> Please be thorough about what to test to help reviewers. You might want to emphasize potential regressions to check for, if your code relies on a feature flag for checking both paths of the feature flag, other parts of the code that may have been impacted by your changes, etc.

Reviewer: ```suggestion
/* Please be thorough about what to test to help reviewers. You might want to emphasize potential regressions to check for, if your code relies on a feature flag for checking both paths of the feature flag, other parts of the code that may have been impacted by your changes, etc. */
```

- [ ] Added labels?
- [ ] Added e2e tests? if not please specify why
- [ ] If you added new files, did you update the CODEOWNERS file?

> - [ ] If your changes are visula, did you check both light and dark theme?

Reviewer 2: ```suggestion
- [ ] If your changes are visual, did you check both the light and dark themes?
```
